applications/search/src/pages/api-details-page/listType1/twoColRow/twoColRow.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const TwoColRow = props => {
```

Comment:
dette ser ut som generel komponent. kan dette være i components?

Reply:
moved to components